cmd/vela-server/schedule.go

```diff
@@ -240,7 +240,7 @@ func processSchedule(ctx context.Context, s *library.Schedule, compiler compiler
 		pipeline, err = database.GetPipelineForRepo(context.TODO(), b.GetCommit(), r)
```

Comment:
should this TODO?

Reply:
nope, oversight from a previous PR good catch